src/cljs/rems/administration/create_form.cljs

```diff
@@ -8,14 +8,15 @@
             [rems.administration.items :as items]
             [rems.atoms :as atoms :refer [document-title]]
             [rems.collapsible :as collapsible]
-            [rems.common.form :refer [field-visible? generate-field-id]]
+            [rems.common.form :refer [field-visible? generate-field-id validate-form] :as common-form]
```

Comment:
hmm is create_form.cljs prepared to receive validation errors from the backend? it shouldn't be possible of course now that you use the same validation code on both ends but it might still happen if there's a bug or e.g. the set of languages is different for some reason

Reply:
I added an idea about this as well. Seems like we can fix it later if it doesn't work.